panic: opengl: width (16384) must be less than or equal to 8192

Running the latest version of OpenDiablo2 at c51e46467640290c8023843d6eb682e735b88bdc foo, the following errors is encountered during the first loading screen.

Note, the issue is not present in rev a105bb390a8641b8b5b2c024b6e65d2c8b775a5b, so the commit that introduced the issue is 7859b69da60d60851e2e8e41d2f2aabd0797d162.
```
u@x220 ~/D/OpenDiablo2> OpenDiablo2
2019/11/09 18:14:02.009272 /home/u/Desktop/OpenDiablo2/main.go:32: OpenDiablo2 - Open source Diablo 2 engine
2019/11/09 18:14:02.014032 /home/u/Desktop/OpenDiablo2/core/Engine.go:78: Loading configuration file
2019/11/09 18:14:02.083504 /home/u/Desktop/OpenDiablo2/common/Palette.go:47: Loaded 18 palettes
2019/11/09 18:14:02.114719 /home/u/Desktop/OpenDiablo2/common/TextDictionary.go:35: Loaded 9094 entries from the string table
2019/11/09 18:14:02.115934 /home/u/Desktop/OpenDiablo2/common/LevelTypes.go:50: Loaded 38 LevelType records
2019/11/09 18:14:02.122797 /home/u/Desktop/OpenDiablo2/common/LevelPresets.go:88: Loaded 1091 level presets
2019/11/09 18:14:02.124343 /home/u/Desktop/OpenDiablo2/common/LevelWarp.go:48: Loaded 83 level warps
2019/11/09 18:14:02.127621 /home/u/Desktop/OpenDiablo2/common/ObjectTypes.go:30: Loaded 573 object types
2019/11/09 18:14:02.142292 /home/u/Desktop/OpenDiablo2/common/Objects.go:352: Loaded 571 objects
2019/11/09 18:14:02.149965 /home/u/Desktop/OpenDiablo2/common/Weapons.go:284: Loaded 307 weapons
2019/11/09 18:14:02.153579 /home/u/Desktop/OpenDiablo2/common/Armor.go:277: Loaded 203 armors
2019/11/09 18:14:02.157234 /home/u/Desktop/OpenDiablo2/common/UniqueItems.go:132: Loaded 352 unique items
2019/11/09 18:14:02.177851 /home/u/Desktop/OpenDiablo2/common/Missiles.go:304: Loaded 684 missiles
2019/11/09 18:14:02.199542 /home/u/Desktop/OpenDiablo2/common/Sounds.go:102: Loaded 4699 sound definitions
2019/11/09 18:14:02.212278 /home/u/Desktop/OpenDiablo2/common/ObjectLookup.go:92: Loaded 7892 object lookups
2019/11/09 18:14:02.227565 /home/u/Desktop/OpenDiablo2/common/AnimationData.go:47: Loaded 3529 animation data records
panic: opengl: width (16384) must be less than or equal to 8192

goroutine 23 [running]:
github.com/hajimehoshi/ebiten/internal/graphicsdriver/opengl.(*Driver).checkSize(0xba83e0, 0x4000, 0x20)
	/home/u/goget/pkg/mod/github.com/hajimehoshi/ebiten@v1.10.1-0.20191108205544-35436ea50457/internal/graphicsdriver/opengl/driver.go:62 +0x201
github.com/hajimehoshi/ebiten/internal/graphicsdriver/opengl.(*Driver).NewImage(0xba83e0, 0x2300, 0x1e, 0x1, 0x0, 0x0, 0x0)
	/home/u/goget/pkg/mod/github.com/hajimehoshi/ebiten@v1.10.1-0.20191108205544-35436ea50457/internal/graphicsdriver/opengl/driver.go:77 +0xaa
github.com/hajimehoshi/ebiten/internal/graphicscommand.(*newImageCommand).Exec(0xc0010ee100, 0x156, 0x0, 0x0)
	/home/u/goget/pkg/mod/github.com/hajimehoshi/ebiten@v1.10.1-0.20191108205544-35436ea50457/internal/graphicscommand/command.go:563 +0x50
github.com/hajimehoshi/ebiten/internal/graphicscommand.(*commandQueue).Flush(0xba81c0)
	/home/u/goget/pkg/mod/github.com/hajimehoshi/ebiten@v1.10.1-0.20191108205544-35436ea50457/internal/graphicscommand/command.go:272 +0x780
github.com/hajimehoshi/ebiten/internal/graphicscommand.FlushCommands(...)
	/home/u/goget/pkg/mod/github.com/hajimehoshi/ebiten@v1.10.1-0.20191108205544-35436ea50457/internal/graphicscommand/command.go:305
github.com/hajimehoshi/ebiten/internal/restorable.ResolveStaleImages()
	/home/u/goget/pkg/mod/github.com/hajimehoshi/ebiten@v1.10.1-0.20191108205544-35436ea50457/internal/restorable/images.go:55 +0x2d
github.com/hajimehoshi/ebiten/internal/shareable.EndFrame(0xc000339980, 0xc001112040)
	/home/u/goget/pkg/mod/github.com/hajimehoshi/ebiten@v1.10.1-0.20191108205544-35436ea50457/internal/shareable/image.go:562 +0x3b
github.com/hajimehoshi/ebiten/internal/buffered.EndFrame(...)
	/home/u/goget/pkg/mod/github.com/hajimehoshi/ebiten@v1.10.1-0.20191108205544-35436ea50457/internal/buffered/image.go:43
github.com/hajimehoshi/ebiten.(*uiContext).Update(0xc000339880, 0xc002423d00, 0x0, 0x0)
	/home/u/goget/pkg/mod/github.com/hajimehoshi/ebiten@v1.10.1-0.20191108205544-35436ea50457/uicontext.go:130 +0x307
github.com/hajimehoshi/ebiten/internal/uidriver/glfw.(*UserInterface).update(0xb6c540, 0x943fa0, 0xc000339880, 0x0, 0x0)
	/home/u/goget/pkg/mod/github.com/hajimehoshi/ebiten@v1.10.1-0.20191108205544-35436ea50457/internal/uidriver/glfw/ui.go:793 +0x1d6
github.com/hajimehoshi/ebiten/internal/uidriver/glfw.(*UserInterface).loop(0xb6c540, 0x943fa0, 0xc000339880, 0x0, 0x0)
	/home/u/goget/pkg/mod/github.com/hajimehoshi/ebiten@v1.10.1-0.20191108205544-35436ea50457/internal/uidriver/glfw/ui.go:821 +0x8a
github.com/hajimehoshi/ebiten/internal/uidriver/glfw.(*UserInterface).run(0xb6c540, 0x320, 0x258, 0x3ff0000000000000, 0xc00082c020, 0x1a, 0x943fa0, 0xc000339880, 0x40000, 0x40000)
	/home/u/goget/pkg/mod/github.com/hajimehoshi/ebiten@v1.10.1-0.20191108205544-35436ea50457/internal/uidriver/glfw/ui.go:693 +0x165
github.com/hajimehoshi/ebiten/internal/uidriver/glfw.(*UserInterface).Run.func1(0xc000012220, 0xc0000a4420, 0xb6c540, 0x320, 0x258, 0x3ff0000000000000, 0xc00082c020, 0x1a, 0x943fa0, 0xc000339880)
	/home/u/goget/pkg/mod/github.com/hajimehoshi/ebiten@v1.10.1-0.20191108205544-35436ea50457/internal/uidriver/glfw/ui.go:573 +0xdf
created by github.com/hajimehoshi/ebiten/internal/uidriver/glfw.(*UserInterface).Run
	/home/u/goget/pkg/mod/github.com/hajimehoshi/ebiten@v1.10.1-0.20191108205544-35436ea50457/internal/uidriver/glfw/ui.go:570 +0x1c8
```

--- a/common/Sprite.go
+++ b/common/Sprite.go
@@ -141,13 +141,13 @@ func CreateSprite(data []byte, palette PaletteRec) *Sprite {
 	totalHeight := 0
 	frame := 0
 	for d := 0; d < int(result.Directions); d++ {
-		curMaxHeight := 0
+		curMaxWidth := 0
 		for f := 0; f < int(result.FramesPerDirection); f++ {
-			totalWidth += int(result.Frames[frame].Width)
-			curMaxHeight = int(Max(uint32(curMaxHeight), result.Frames[frame].Height))
+			curMaxWidth = int(Max(uint32(curMaxWidth), result.Frames[frame].Width))
+			totalHeight += int(result.Frames[frame].Height)
 			frame++
 		}
-		totalHeight += curMaxHeight
+		totalWidth += curMaxWidth
 	}
 	result.atlas, _ = ebiten.NewImage(totalWidth, totalHeight, ebiten.FilterNearest)
 	result.atlasBytes = make([]byte, totalWidth*totalHeight*4)
@@ -155,15 +155,15 @@ func CreateSprite(data []byte, palette PaletteRec) *Sprite {
 	curX := 0
 	curY := 0
 	for d := 0; d < int(result.Directions); d++ {
-		curMaxHeight := 0
+		curMaxWidth := 0
 		for f := 0; f < int(result.FramesPerDirection); f++ {
-			curMaxHeight = int(Max(uint32(curMaxHeight), result.Frames[frame].Height))
+			curMaxWidth = int(Max(uint32(curMaxWidth), result.Frames[frame].Width))
 			result.Frames[frame].Image = result.atlas.SubImage(image.Rect(curX, curY, curX+int(result.Frames[frame].Width), curY+int(result.Frames[frame].Height))).(*ebiten.Image)
-			curX += int(result.Frames[frame].Width)
+			curY += int(result.Frames[frame].Height)
 			frame++
 		}
-		curY += curMaxHeight
-		curX = 0
+		curX += curMaxWidth
+		curY = 0
 	}
 	return result
 }

